### PR TITLE
Get branch up to date with development

### PR DESCRIPTION
**Pull request by KrisVan**

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -1,9 +0,0 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
-
-
-test('renders title', () => {
```
Comment (drew28) [inline src/App.test.js:5]: why did you delete this? this is a unit test where you can test component specifics

the playwright tests are considered more e2e (end-to-end) or functional tests

it's good to have both and know the difference between the 2

Comment (KrisVan) [reply src/App.test.js:5]: Misconception between unit and e2e testing. Thought that they were redundant with the playwright tests.

Added the unit tests back in 71315a0
